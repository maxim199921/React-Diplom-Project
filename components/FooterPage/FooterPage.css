.wrapper {
    display: flex;
    flex-direction: column;
    height: 100px/*100vh*/;
    color: white;
}

.mainLink {
    color: black;
    flex: 1;
}

.footerLink {
    background: #130e27;
    width: 100%;
    min-height: 60px;
    text-align: center;
    font-size: 20px;
    line-height: 22px;
    color: #5dbbc1;
    font-style: italic;
    font-weight: bold;
}

.footer-wrapper {
    min-height: 60px;
}

.footerLink-logo {
    background-image: url("../../img/LogoFooter.png");
    background-repeat: no-repeat;
    background-position: right;
    width: 120px;
    height: 55px;
}

.footerLink-info {
    text-align: left;
}

.footerLink-adress {
    border: 1px solid white;
}

@media (max-width: 767px) {
    .footerLink-logo {
        display: none;
    }
    .footerLink-info {
        text-align: center;
    }
}