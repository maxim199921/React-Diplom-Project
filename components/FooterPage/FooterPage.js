import React from 'react';

import './FooterPage.css';

class FooterPage extends React.PureComponent {

    render() {

        return (
            <div className="footerLink">
                <div className="container">
                    <div className="row no-gutters align-items-center footer-wrapper">
                        <div className="col-12 col-md-8 footerLink-info">Developed by: M.C.Chupilin</div>
                        <div className="col-md-4 footerLink-logo"/>
                    </div>
                </div>
            </div>
        );
    }

}

export default FooterPage;
