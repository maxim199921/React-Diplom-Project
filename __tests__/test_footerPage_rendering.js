import React from 'react';
import renderer from 'react-test-renderer';
import FooterPage from '../components/FooterPage/FooterPage';


test('тест на рендеринг футера', () => {

    const component = renderer.create(
        <FooterPage/>
    );

    let componentTree=component.toJSON();
    expect(componentTree).toMatchSnapshot();

});