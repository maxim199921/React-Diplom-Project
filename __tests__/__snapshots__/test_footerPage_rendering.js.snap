// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`тест на рендеринг футера 1`] = `
<div
  className="footerLink"
>
  <div
    className="container"
  >
    <div
      className="row no-gutters align-items-center footer-wrapper"
    >
      <div
        className="col-12 col-md-8 footerLink-info"
      >
        Developed by: M.C.Chupilin
      </div>
      <div
        className="col-md-4 footerLink-logo"
      />
    </div>
  </div>
</div>
`;
